features.yaml

```diff
@@ -0,0 +1,55 @@
+features:
+  - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
+    level:
+      - checklist:
+        maturity: Stable
+        nextExpectedPromotion:
+    deprecation:
+     - state: active
+       next:
+       details:
+  - name: "Protocols:Websockets/MongoDB"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+    deprecation:
+     - state: active
+       next:
+       details:
+  - name: "Traffic Control: label/content based routing, traffic shifting"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+    deprecation:
+     - state: active
+       next:
+       details:
+  - name: "Gateway: Ingress, Egress for all protocols"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+    deprecation:
+     - state: active
+       next:
+       details:
+  - name: "Gateway: Ingress, Egress for all protocols"
+    level:
+     - checklist:
+       maturity: Stable
```

Comment:
One thing that can be confusing here is the gateway API isn't stable, only beta.  Should we make some comment that this is only for functionality, refer to API proto for API feature status?

Reply:
Hmmm. I pulled that from https://istio.io/latest/about/feature-stages/. I think clarification would be good.

Reply:
@lizan @costinm @PiotrSikora I think Lin brings up a good point here. If the API isn't stable, should we rename the feature on the feature status page to be less ambigous? I'll update the feature stages page as well, but we should use the same title for both. Thoughts on `Gateway Functionality: stable` with another row saying `Gateway API: beta`?

Reply:
I like what you have as 2 rows, functionality vs API.  This will lever set with users well.  so much confusing around this from users.